samd21/usb/class/msc/device/mscdf.c

```diff
@@ -664,6 +692,9 @@ int32_t mscdf_register_callback(enum mscdf_cb_type cb_type, FUNC_PTR func)
 	case MSCDF_CB_GET_DISK_CAPACITY:
 		mscdf_get_disk_capacity = (mscdf_get_disk_capacity_t)func;
 		break;
+	case MSCDF_CB_IS_WRITABLE:
+		mscdf_is_writeable = (mscdf_is_writeable_t)func;
```

Comment:
Two spellings of writable here: https://m-w.com and the online OED only give "writable" as the spelling (don't list the other as a variant.